templates/node--funding--full.html.twig

```diff
@@ -0,0 +1,211 @@
+
+{%
+set classes = [
+'node',
+'node--type-' ~ node.bundle|clean_class,
+node.isPromoted() ? 'node--promoted',
+node.isSticky() ? 'node--sticky',
+not node.isPublished() ? 'node--unpublished',
+view_mode ? 'node--view-mode-' ~ view_mode|clean_class,
+'clearfix',
+]
+%}
+{{ attach_library('classy/node') }}
+{{ attach_library('unl_fourone/funding') }}
+
+{%
+set full_description, funding_amount, funding_periods, application_due_date, contact_name, contact_phone_number, contact_email, eligibility, how_to_apply, templates, resources_documents, resources_external_links, apply_link, application_form, applicable_to =
+    content.field_funding_full_description, content.field_funding_amount, content.field_funding_periods, content.field_funding_app_due_date, content.field_funding_contact_name, content.field_funding_contact_phone, content.field_funding_contact_email, content.field_funding_eligibility, content.field_funding_how_to_apply, content.field_funding_templates, content.field_funding_resources_docs, content.field_funding_resources_links, content.field_funding_apply_link, content.field_funding_application_form, content.field_applicable_to
+%}
+{#to get striptags filter to work on date value#}
+{% set application_due_date_value %}
+    {{ content.field_funding_app_due_date }}
+{% endset %}
+
+<article{{ attributes.addClass(classes) }}>
+
+    {{ title_prefix }}
+    {% if not page and view_mode != "full" %}
+        <h2{{ title_attributes }}>
+            <a href="{{ url }}" rel="bookmark">{{ label }}</a>
+        </h2>
+    {% endif %}
+    {{ title_suffix }}
+
+    {% if display_submitted %}
+        <footer>
+            {{ author_picture }}
+            <div{{ author_attributes }}>
+                {% trans %}Submitted by {{ author_name }} on {{ date }}{% endtrans %}
+                {{ metadata }}
+            </div>
+        </footer>
+    {% endif %}
+
+    <div class="wdn-grid-set">
+        <div class="bp768-wdn-col-three-eighths wdn-pull-right">
+            {# taxonomy section that is visible in mobile #}
+            <section class="funding-ct--taxonomy-tags-section--mobile">
+                {% if node.field_applicable_to.value %}
+                    <span class="wdn-text-hidden">{{ applicable_to['#title'] }}</span>
+                    <ul class="funding-ct--taxonomy-tags">
+                        {% for i, value in applicable_to['#items'] %}
+                            {% set tag_value = applicable_to[i]['#title']|split(':')%}
```

Comment:
I /think/ you could just use a `for` loop with the magic `loop.index0` variable to get the current index too. I don't think you need to change this, I just thought I'd mention it.